test/validators/SignedKeyRequestValidator/SignedKeyRequestValidator.t.sol

```diff
@@ -0,0 +1,223 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.21;
+
+import {SignedKeyRequestValidator} from "../../../src/validators/SignedKeyRequestValidator.sol";
+import {SignedKeyRequestValidatorTestSuite} from "./SignedKeyRequestValidatorTestSuite.sol";
+
+contract SignedKeyRequestValidatorTest is SignedKeyRequestValidatorTestSuite {
+    event SetIdRegistry(address oldIdRegistry, address newIdRegistry);
+
+    function testMetadataTypeHash() public {
+        assertEq(
+            validator.metadataTypehash(), keccak256("SignedKeyRequest(uint256 requestFid,bytes key,uint256 deadline)")
+        );
+    }
+
+    /*//////////////////////////////////////////////////////////////
+                              VALIDATION
+    //////////////////////////////////////////////////////////////*/
+
+    function testFuzzValidate(
+        uint256 signerPk,
```

Comment:
nit: signerPk sounds like it should be the counterpart to signerPubKey. is requesterPk more accurate here?